public/robots.txt

```diff
@@ -1,3 +1,4 @@
 # https://www.robotstxt.org/robotstxt.html
```

Comment:
Also try removing this

Reply:
https://github.com/smaranjitghose/doc2pen/pull/1008/commits/0d164230a69e4a7bc04fc134664109c9150cf758